site.yml

```
@@ -21,460 +21,460 @@ Documents:
+        SEO Keywords: pre-signed URLs
 
       - How to run Minio in FreeNAS?:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/running-minio-in-freenas.md
         Slug: how-to-run-minio-in-freenas
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to run MinIO in FreeNAS? - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to run MinIO with FreeNAS.
+        SEO Keywords: FreeNAS
 
       - How to use AWS SDK for PHP with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-php-with-minio.md
         Slug: how-to-use-aws-sdk-for-php-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for PHP with MinIO Server - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to use aws-sdk-php, the official AWS SDK for the PHP programming language, with MinIO server.
+        SEO Keywords: AWS SDK for PHP
 
       - How to use AWS SDK for Ruby with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-ruby-with-minio.md
         Slug: how-to-use-aws-sdk-for-ruby-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for Ruby with MinIO Server - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to use aws-sdk, the official AWS SDK for the Ruby programming language, with MinIO server
+        SEO Keywords: AWS SDK for Ruby, AWS SDK for Ruby on Rails
 
       - How to use AWS SDK for Python with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-python-with-minio.md
         Slug: how-to-use-aws-sdk-for-python-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for Python with MinIO Server - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to use aws-sdk-python, the official AWS SDK for the Python programming language, with MinIO server.
+        SEO Keywords: AWS SDK for Python
 
       - How to use AWS SDK for JavaScript with Minio Server:
         Link: https://raw.githubusercontent.com/minio/cookbook/master/docs/aws-sdk-for-javascript-with-minio.md
         Slug: how-to-use-aws-sdk-for-javascript-with-minio-server
-        SEO Title:
-        SEO Description:
-        SEO Keywords:
+        SEO Title: MinIO | How to use AWS SDK for JavaScript with MinIO Server - Cookbook/Recipe
+        SEO Description: In this recipe we will learn how to use aws-sdk for JavaScript with MinIO server. aws-sdk is the official AWS SDK for the JavaScript programming language.
```

Can we make this change here to keep the descriptions of the AWS SDK guides consistent with each other?

```suggestion
        SEO Description: In this recipe we will learn how to use aws-sdk, the official AWS SDK for the JavaScript programming language, with MinIO server.
```